deps/exokit-bindings/webglcontext/src/webgl.cc

```diff
@@ -4467,6 +4464,18 @@ NAN_METHOD(WebGLRenderingContext::GetParameter) {
       info.GetReturnValue().Set(JS_INT(param));
       break;
     }
+    case GL_MAX_VARYING_COMPONENTS:
```

Comment:
`gl.getParameter(gl.MAX_VARYING_COMPONENTS)` was giving an error, but according to the spec `MAX_VARYING_COMPONENTS` is always identical to `4*MAX_VARYING_VECTORS`, and `gl.getParameter(gl.MAX_VARYING_VECTORS)` seems to work fine.

Reply:
Which operating system/GPU driver was this tested on? If the GL spec says `GL_MAX_VARYING_COMPONENTS` exists, then we should probably query that.